screens/WelcomeScreen.py

```diff
@@ -0,0 +1,35 @@
+from screens.screen import Screen
+import pygame_gui
+import pygame
+from main import nav_to_question_screen
+
+class WelcomeScreen(Screen):
+    def __init__(self, screen_size, clock, loader):
+        super().__init__(screen_size, clock, loader)
+        self.init_ui()
```

Comment:
Add self.set_callback before line 9 and set the set_callback constructor parameter to its value.